src/h3lib/include/h3api.h.in

```diff
@@ -260,6 +275,16 @@ double H3_EXPORT(hexAreaKm2)(int res);
 
 /** @brief average hexagon area in square meters (excludes pentagons) */
 double H3_EXPORT(hexAreaM2)(int res);
+
+/** @brief exact area for a specific cell (hexagon or pentagon) in radians^2 */
+double H3_EXPORT(cellAreaRads2)(H3Index h);
```

Comment:
I assume for these functions they have undefined output when the H3 index is invalid.

Reply:
yes, should we document that? or test for validity?